examples/03_LLD_CoRh2O4.jl

```diff
@@ -46,6 +50,11 @@ for _ in 1:1000
 end
 plot(energies, color=:blue, figure=(size=(600,300),), axis=(xlabel="Time steps", ylabel="Energy (meV)"))
 
+# After running a Langevin trajectory, it is a good practice to call
```

Comment:
Asked about this on slack. It seems reasonable that a trajectory should be collected for statistics. I guess there are some question about whether this implicit definition of a trajectory is the best interface. Will comment more outside of code.